packages/addons/service/prometheus-node-exporter/source/resources/settings.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings>
+	<category label="30000">
+		<setting id="WebListenPort"             label="30001" type="number" default="9100"     />
```

Comment:
pls convert tabs -> spaces

Reply:
Yes , done.

Creating files with vscode and vim , maybe that :)